src/alto-ui/Form/DatePicker/DatePicker.js

```diff
@@ -126,7 +125,17 @@ class DatePicker extends React.Component {
   }
 
   render() {
-    const { id, small, large, placeholder } = this.props;
+    const {
+      id,
+      small,
+      large,
+      placeholder,
+      error,
+      readOnly,
+      label,
+      hideLabel,
+      helpText,
+    } = this.props;
```

Comment:
No you should do the opposite, extract datepicker props and spread the remaining in Textfield.

```js
const { onChange, iso, timestamp, format ...inputProps } = this.props;
```